lib/ceremony.hoon

```diff
@@ -207,7 +215,9 @@
   =.  network
     ?+  net  0x1
       %ropsten  0x3
+      %fake     `@ux``@`1.337
```

Comment:
Local ganache fakenode uses `0x1` always, as far as I'm aware. Is that configurable?